### PR TITLE
Update Unova route encounters

### PR DESCRIPTION
**Pull request by Ultima1990**
Added some encounters to Unova routes. Removed some. Took some creative liberty and put Alomomola on 17 and 18.

Also added red/blue Basculin and Deerling+Sawsbuck season forms to allow for the new route additions.

### CODE REVIEW

**Comment by RedSparr0w** [comment]
Haven't had a chance to fully look over this, but with the new forms, could you also add the normal & shiny images.

They should be `96 x 96px` `.png` and run them through something like https://tinypng.com too.

**Comment by Ultima1990** [comment]
It has been done.

File: src/scripts/wildBattle/Routes.ts [hunk 1]
```diff
@@ -1188,49 +1194,54 @@ Routes.add(new RegionRoute(
...
-        land: ['Frillish'],
+        land: ['Frillish', 'Alomomola'],
     }),
     [new RouteKillRequirement(10, 1)]
 ));
 Routes.add(new RegionRoute(
     GameConstants.Region.unova, 18,
     new RoutePokemon({
-        land: ['Scrafty', 'Crustle'],
+        land: ['Tropuis', 'Carnivine', 'Watchog', 'Crustle', 'Scrafty'],
```
Comment (RedSparr0w) [inline src/scripts/wildBattle/Routes.ts:1204]: `Tropuis` shouldn't be here

**Review by RedSparr0w — APPROVED** [review]
👍 Changes look good,

Thanks for the PR!

**Pull request merged**